mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/MongoDBDialect.java

```diff
@@ -849,8 +849,13 @@ public void forEachTuple(ModelConsumer consumer, TupleTypeContext tupleTypeConte
 		MongoDBQueryDescriptor queryDescriptor = backendQuery.getQuery();
 
 		EntityKeyMetadata entityKeyMetadata =
-				backendQuery.getSingleEntityMetadataInformationOrNull() == null ? null :
-					backendQuery.getSingleEntityMetadataInformationOrNull().getEntityKeyMetadata();
+			( backendQuery.getSingleEntityMetadataInformationOrNull() == null ) ? null :
+				backendQuery.getSingleEntityMetadataInformationOrNull().getEntityKeyMetadata();
```

Comment:
It's not critical, but I prefer this format for this:

```
		backendQuery.getSingleEntityMetadataInformationOrNull() == null
                                ? null
				: backendQuery.getSingleEntityMetadataInformationOrNull().getEntityKeyMetadata();
```

I find it more readable